src/component/ListBox.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import TodoItem from "./TodoItem"
+import styled, { css } from "styled-components";
+
+const List = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 200px;
+  overflow: hidden auto;
+  ${({ type }) =>
+    type == "done" &&
+    css`
+      text-decoration: line-through;
+    `}
```

Comment:
이부분은 아래와 같이 바꾸면 좀 더 깔끔할 것 같아요
```suggestion
  text-decoration: ${(props) => props.type === 'done' ?? 'line-through' : 'none';
```

Reply:
오오.. 이 부분은 다른 레퍼런스를 참고해서 작성했던 코드였는데, 더 깔끔하게 써볼 수는 없을까 많이 고민했던 부분이었습니다! 문기님 말씀대로 수정을 한다면 너무 깔끔해지겠네요 이렇게 수정해봐야겠어요! 감사합니다:D

문기님 코드에서도 봤지만 `props` 너무 깔끔하게 잘 쓰시네요 최고입니다😄